hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -411,7 +411,7 @@ public Response head(
 
     OzoneKey key;
     try {
-      key = getBucket(bucketName).headObject(keyPath);
```

Comment:
This call is present in other places as well (e.g get, put, delete, abort etc). Is it possible to fix there as well?

Reply:
Yup, it's definitely what we want. I'd like to take that review to confirm the approach then we can apply that for other S3 APIs.